config/settings_ts/README.md

```diff
@@ -0,0 +1 @@
+Sample settings for Testing Stand.
```

Comment:
I don't think we need to commit actual config values into main repo as it has nothing to do with salmon functionality.

I guess, it's better to commit those into testing-stand repo as we need to share these "canonical" configs somehow.